plugin/aws/ec2Instance.go

```diff
@@ -225,7 +222,24 @@ func (inst *ec2Instance) Exec(ctx context.Context, cmd string, args []string, op
 			identityfile = (filepath.Join(homedir, ".ssh", (keyname.(string) + ".pem")))
 		}
 	}
-	// Use the default user for Amazon AMIs. See above for ideas on making this more general. Can be
-	// overridden in ~/.ssh/config.
-	return transport.ExecSSH(ctx, transport.Identity{Host: hostname, FallbackUser: "ec2-user", IdentityFile: identityfile}, append([]string{cmd}, args...), opts)
+	var fallbackuser string
```

Comment:
I'd personally appreciate a newline to separate getting the fallbackuser from getting identityfile.

Reply:
Done